### ISSUE TITLE
Proposal: semver-minor and semver-patch labels

### ISSUE THREAD

**Issue by not-an-aardvark** [eslint/eslint#7498]
At the moment, we use the `enhancement` and `feature` labels for any semver-minor change. However, according to our [semver policy](https://github.com/eslint/eslint/blob/master/README.md#semantic-versioning-policy), not all semver-minor changes are enhancements; for example, fixing a false negative in a rule is a semver-minor change. We currently put the `enhancement` label on false-negative bugfixes to clarify that they're semver-minor, but I think this is mildly confusing; a false-negative bugfix isn't any more of an "enhancement" than a false-positive bugfix is. IMO, we're overloading the "enhancement" label to mean "semver-minor".

As a solution, I think we should add `semver-minor` and `semver-patch` labels. This would make it easier to see what can be merged for a patch release, and we would be able to use the `enhancement` label only for actual enhancements, such as new rule options.

---

I think our use `Fix:` and `Update:` in commit messages has the same problem; false-negative fixes use the `Update:` label to indicate that they're semver-minor. For example, some commit messages look like this:

```
Update: fix false negative of foo
```

But I think messages like this are easier to understand:

```
Fix: false negative of foo
```

The second commit message makes it easier for anyone skimming the changelog for new features (since they can just look at all commits with `Update:` and `New:` prefixes). At the moment, we use the `Update:` prefixe for bugfixes as well, which is a bit misleading to the reader.

(However, I know that these prefixes are used by the release tool to determine what kind of release to create, so I don't have a good solution for this problem aside from using separate prefixes for different types of bugfixes.)

**Comment by kaicataldo** [comment]
My initial reaction to this was that it seemed like a fine thing to add labels, but now that I'm seeing the interaction with prefixes, it does seem a bit more complicated (and more work) than I originally thought. It sounds like the changes would cascade into changes to the bot, documentation, and retraining ourselves and our contributors to use the new labels correctly - energy that might better serve the project if we can focus on things like JSCS compatibility.

I'm not against it being clearer, but I don't think we've run into any real issues because of this ambiguity. Happy to discuss this, though, and see if we can make it better!

**Comment by not-an-aardvark** [comment]
There doesn't seem to be much interest in adding these labels, so I'll close this issue.
